dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ParseConfigUtils.java

```diff
@@ -40,32 +42,56 @@
 
   public static final Map<String, Object> flatItems = new HashMap<>();
 
+  public static String CURRENT_VERSION_INFO = "default";
```

Comment:
1. Do not use public static field. 
2. Non final field should be lower case. Only final field be upper case.